<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditInstructionDialog</class>
 <widget class="QDialog" name="EditInstructionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>321</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="instructionTypeComboBox">
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Data handling</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Control flow</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Call</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Arithmetic/Logic</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Privileged</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="instructionPrivilegedComboBox">
         <item>
          <property name="text">
           <string>Unknown</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Description (basic HTML tags allowed)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="instructionDescriptionTextEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditInstructionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditInstructionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
